dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -76,7 +76,7 @@ def summarize_progress
     # If the course unit has an evaluation level, then status is determined by the completion of the focus group modules
     if plc_course_unit.has_evaluation?
       Plc::LearningModule::MODULE_TYPES.select {|type| categories_for_stage.include?(type)}.each do |flex_category|
-        module_category = flex_category || Plc::LearningModule::CONTENT_MODULE
+        module_category = flex_category
```

Comment:
There are no falsy values in the `MODULE_TYPES` constant. Therefore, we will never fall back on this or-case.